src/Donors/DataTransferObjects/DonorResponseData.php

```diff
@@ -88,7 +89,7 @@ public static function fromObject($donor)
         $self->titlePrefix = $donor->titlePrefix;
         $self->donationCount = (int)$donor->donationCount;
         $self->dateCreated = Date::getDateTime($donor->createdAt);
-        $self->donorType = give()->donors->getDonorType($donor->id);
+        $self->donorType = give()->donors->getDonorType($donor->id)->getValue();
```

Comment:
Is this class being used anymore? If not we can delete it.

Reply:
Resolved: 695c09379623f0130f7c306df046305f3b51a77d